lib/index.js

```diff
@@ -48,14 +48,27 @@ function getRepo(options) {
  * Push a git branch to a remote (pushes gh-pages by default).
  * @param {string} basePath The base path.
  * @param {Object} config Publish options.
- * @param {Function} done Callback.
+ * @param {Function} callback Callback.
  */
-exports.publish = function publish(basePath, config, done) {
+ /*eslint-disable no-console */
+exports.publish = function publish(basePath, config, callback) {
   if (typeof config === 'function') {
-    done = config;
+    callback = config;
     config = {};
   }
 
+  var done = function(arguments) {
+    try {
+      (callback || function(err) {
+        if (!err) {
+          console.log(err)
```

Comment:
and `err != null` was linted with this error: `gh-pages/lib/index.js
  63:13  error  Use ‘===’ to compare with ‘null’  no-eq-null`

which is really sad, because null check with `!=` makes sense to check against `undefined` and `null`